app/src/main/java/com/teamwss/websoso/ui/activityDetail/ActivityDetailViewModel.kt

```diff
@@ -2,35 +2,85 @@ package com.teamwss.websoso.ui.activityDetail
 
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.SavedStateHandle
 import androidx.lifecycle.ViewModel
-import com.teamwss.websoso.data.repository.UserFeedRepository
+import androidx.lifecycle.viewModelScope
+import com.teamwss.websoso.data.repository.UserRepository
+import com.teamwss.websoso.ui.activityDetail.ActivityDetailActivity.Companion.SOURCE_MY_ACTIVITY
 import com.teamwss.websoso.ui.main.myPage.myActivity.model.ActivitiesModel.ActivityModel
+import com.teamwss.websoso.ui.mapper.toUi
 import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.launch
 import javax.inject.Inject
 
 @HiltViewModel
 class ActivityDetailViewModel @Inject constructor(
-    private val activityRepository: UserFeedRepository,
+    private val userRepository: UserRepository,
+    private val savedStateHandle: SavedStateHandle,
 ) : ViewModel() {
 
     private val _userActivity = MutableLiveData<List<ActivityModel>>()
     val userActivity: LiveData<List<ActivityModel>> get() = _userActivity
 
-    init {
-//        updateMyActivities()
+    private val _lastFeedId =  MutableLiveData<Long>(0L)
+    val lastFeedId: LiveData<Long> get() = _lastFeedId
+
+    private val size: Int = ACTIVITY_SIZE
```

Comment:
```suggestion
    private val activityLoadSize: Int = ACTIVITY_LOAD_SIZE
```
c: 변수명이 애매한거 같은데  이렇게 수정해주는건 어떤지, 사이즈라고 그냥하면 뭔지 모르니까

Reply:
넵 수정하겠습니다!!